distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -441,7 +359,7 @@ type Array
 
             [[1, 2, 3].to_array, [4, 10].to_array, [].to_array, [0].to_array, [0].to_array].to_array . flatten == [1, 2, 3, 4, 10, 0, 0].to_array
     flatten : Vector Any
-    flatten self = Vector.flatten self
+    flatten self = (Vector.from_polyglot_array self).flatten
```

Comment:
That seems like a weird restriction. I'd rather we keep the call using the static wrapper and add a specialization to `flatten`, unless it is not possible.